src/components/interaction-designer/block-editors/BlockOutputBranchingConfig.vue

```diff
@@ -160,7 +160,7 @@
 
       switch (value) {
         case OutputBranchingType.UNIFIED:
-          this.block_convertExitFormationToUnified({blockId})
+          this.$emit('branchingTypeChangedToUnified')
```

Comment:
I'm wondering if we should do both of these so that the wrapping component is responding to the change rather than expecting the wrapper component to commit the change. It feels a bit different than other editors in that it that the wrapper NEEDs to implement something. 

Is there a default scenario where we would want to be able to drop `<BlockOutputBranchingConfig />` in and not have to implement any handler?

Reply:
Then the wrapper component would receive the event and update the test property itself using its own block store? I'm not 100% confident, but what are your thoughts?

Reply:
According what we have discussed, we will leave it as I have implemented.